lib/cog/bundle_registry.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Cog.BundleRegistry do
+  @registry_url "https://bundles.operable.io"
+
+  def get_config(bundle, version) do
+    headers = ["Accepts": "application/json"]
+    response = HTTPotion.get(@registry_url <> "/api/bundles/#{bundle}/#{version}", headers: headers)
+
+    case response do
+      %HTTPotion.Response{status_code: 200, body: body} ->
+        {:ok, Poison.decode!(body)}
+      %HTTPotion.Response{status_code: 404} ->
+        {:error, "Bundle not found"}
```

Comment:
We should probably have a catch-all case for other errors, too (e.g., registry is unavailable, registry crashes, etc.) just so we don't crash Cog.